client/src/assets/images/FreeCodeCampFive.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+const propTypes = {};
+
+function FreeCodeCampFive(props) {
+  return (
+    <svg
+      viewBox='93.907 285.379 4503.093 1366.621'
+      xmlns='http://www.w3.org/2000/svg'
+      xmlnsXlink='http://www.w3.org/1999/xlink'
+      {...props}
+    >
```

Comment:
```suggestion
      fill='var(--secondary-color)'
    >
```
this will keep the color aligned with theme.

Reply:
Hmm, unless something has a stroke as well? 

Where you would need both the color and the background? Should we take the opportunity to create dedicated variables for the SVG fills and Strokes?

Reply:
this one should not be affected. whoever, we should set the stroke as well just to be safe.